docs/auth.md

```diff
@@ -43,26 +233,23 @@ aggregates them into their respective files in `$HOME`.
        tekton.dev/git-0: github.com # Described below
    type: kubernetes.io/ssh-auth
    data:
```

Comment:
We use stringData above to avoid requiring users to base64 encode the data, it would probably be better to use it here as well.

Reply:
Which lines should I change here and to what?

Reply:
All fixed.